TYPO3.Media/Classes/TYPO3/Media/Domain/Service/ThumbnailService.php

```diff
@@ -82,62 +82,28 @@ public function getThumbnail(AssetInterface $asset, ThumbnailConfiguration $conf
     {
         $thumbnail = $this->thumbnailRepository->findOneByAssetAndThumbnailConfiguration($asset, $configuration);
```

Comment:
actually changed that to support the asynchronous generation, will be changed in that PR